Sources/UtilityBeltUITesting/Extensions/ProcessInfo+LaunchArguments.swift

```diff
@@ -0,0 +1,12 @@
+// Copyright © 2021 SpotHero, Inc. All rights reserved.
+
+import Foundation
+
+public extension ProcessInfo {
+    /// Determines whether or not the arguments for the process contain the specific launch argument.
+    /// - Parameter argument: The argument in question.
+    /// - Returns: Whether or not the arguments for the process contain the specific launch argument.
+    static func containsLaunchArgument(_ argument: LaunchArgument) -> Bool {
+        return self.processInfo.arguments.contains(argument.rawValue)
```

Comment:
just a general question - whats up with the `processInfo` instance on the `ProcessInfo` class? Should this be a function on the instance or a static function on the class?

Reply:
The `processInfo` var is the shared instance. Each process has a single, shared `ProcessInfo` object. (https://developer.apple.com/documentation/foundation/processinfo) Since there is only one shared object I opted for a static function for simplicity, when using.

Would you prefer I use an instance function?